schemas/derivatives/SCHEMA.md

```diff
@@ -0,0 +1,83 @@
+# Derivatives
+
+OpenBlock Labs standard schema for derivatives protocols (options and perps).
+
+## Version: 1.0.0-alpha
+
+### Pools
+
+Pools in the protocol (one entry for each token).
+
+| Property                | Description                                               | Type   |
+|-------------------------|-----------------------------------------------------------|--------|
+| chain_id                 | Standard chain id.                                        | number |
+| creation_timestamp       | The timestamp of the block the pool was created on.       | number |
+| creation_block_number    | The block number this pool was created on.                | number |
+| pool_address             | The contract address of the pool.                         | string |
+| pool_name                | The name of the pool (ie, name() in the smart contract).  | string |
+| token_index              | The index of the token in the smart contract (one row for each token in a pool). | number |
+| token_address            | The contract address of the token.                        | string |
+| token_symbol             | The symbol of the token.                                  | string |
+
+### LP Snapshot
+
+Liquidity providers snapshot (one entry for each token in a pool).
+
+| Property                | Description                                               | Type   |
+|-------------------------|-----------------------------------------------------------|--------|
+| timestamp                | The timestamp of the snapshot.                            | number |
+| chain_id                 | Standard chain id.                                        | number |
+| pool_address             | The address of the pool.                                  | string |
+| lp_address               | The address of the liquidity provider.                    | string |
+| token_index              | The index of the token in the smart contract.             | number |
+| token_address            | The address of the token provided as liquidity.           | string |
+| token_symbol             | The symbol of the token.                                  | string |
+| deposit_amount           | (Optional, for Order Books) The amount deposited into the pool, decimal normalized. | number |
+| deposit_amount_usd       | (Optional, for Order Books) The amount deposited, in USD. | number |
+| amount_useful            | (Optional, for LPs) The amount of tokens supplied that are liquid and usable by users of the protocol, decimal normalized. | number |
+| amount_useful_usd        | (Optional, for LPs) The amount of liquid tokens supplied in USD. | number |
+| liquidated_amount        | The amount of tokens liquidated from the LP, decimal normalized. | number |
+| liquidated_amount_usd    | The amount liquidated, in USD.                            | number |
```

Comment:
i think liquidated amount here should be derived from the pool_snapshot table below

Reply:
wouldn't be easier to ask directly for it, to avoid doing this kind of transformation on our side?

Reply:
ok lets leave this as it is